index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main class="container">
+      <div class="todo-app">
+        <h2>To-Do List<img src="images/icon.png" alt="" / class="image"></h2>
+        <div class="row">
+          <input type="text" name="" id="inp_bx" placeholder="Add your Task" />
```

Comment:
As a good practice use same case every where, as an attribute value sometimes you have used kebab case  sometimes snake case, generally as a recommendation use camel case everywhere